p/themes/base-theme/frss.css

```diff
@@ -1282,50 +1282,50 @@ a.website:hover .favicon {
 	position: initial;
 }
 
-.flux .item.title a {
+.flux .flux_header .item.title a {
 	color: var(--frss-font-color-dark);
 	text-decoration: none;
 }
 
-.flux .item.thumbnail {
+.flux .flux_header .item.thumbnail {
 	line-height: 0;
 	padding: var(--frss-padding-top-bottom) var(--frss-padding-flux-items);
 	height: 80px;
 }
 
-.flux .item.thumbnail .item-element {
+.flux .flux_header .item.thumbnail .item-element {
 	padding: 0;
 }
```

Comment:
That was not enough specified and overruled by:
https://github.com/FreshRSS/FreshRSS/blob/354a5e83888cebe89843b754d0936f3633323cef/p/themes/Origine-compact/origine-compact.css#L127-L129